### PR TITLE
KDESKTOP-1132-Change-app-version

### PR DESCRIPTION
**Pull request by ClementKunz**
- Change app version to 3.6.5
- Keep the AUMID from both Windows certificates (EV and virtual) in couldproviderregistrar.cpp in order to simplify future release. Just comment the unnecessary one.

### CODE REVIEW

File: extensions/windows/cfapi/Vfs/cloudproviderregistrar.cpp [hunk 1]
```diff
@@ -37,7 +37,8 @@ using namespace Windows::Security::Cryptography;
 #define REGKEY_AUMID L"AUMID"
 
 // Package family name, see FileExplorerExtensionPackage / Package.appxmanifest
-#define REGVALUE_AUMID L"Infomaniak.kDrive.Extension_csy8f8zhvqa20!App"
+//#define REGVALUE_AUMID L"Infomaniak.kDrive.Extension_dbrs6rk4qqhna!App" // EV
```
Comment (luc-guyot-infomaniak) [inline extensions/windows/cfapi/Vfs/cloudproviderregistrar.cpp:40]: It would be nice to have a comment indicating when this line should be uncommented.

**Review by luc-guyot-infomaniak — COMMENTED** [review]
Comment on a line that is commented out.

File: extensions/windows/cfapi/Vfs/cloudproviderregistrar.cpp (hunk 1 above)
Comment (ClementKunz) [reply extensions/windows/cfapi/Vfs/cloudproviderregistrar.cpp]: Done

File: extensions/windows/cfapi/Vfs/cloudproviderregistrar.cpp [hunk 2]
```diff
@@ -37,7 +37,10 @@ using namespace Windows::Security::Cryptography;
 #define REGKEY_AUMID L"AUMID"
 
 // Package family name, see FileExplorerExtensionPackage / Package.appxmanifest
-#define REGVALUE_AUMID L"Infomaniak.kDrive.Extension_csy8f8zhvqa20!App"
+/// EV (Extended Validation) certificate AUMID : use this certificate to build a release version
+//#define REGVALUE_AUMID L"Infomaniak.kDrive.Extension_dbrs6rk4qqhna!App"
+/// Virtual certificate AUMID : use this certificate in debug mode and to build version for testing purpose
```
Comment (luc-guyot-infomaniak) [inline extensions/windows/cfapi/Vfs/cloudproviderregistrar.cpp:42]:
```suggestion
/// Virtual certificate AUMID : use this certificate in debug mode and to build a version for testing purpose
```

**Review by luc-guyot-infomaniak — COMMENTED** [review]
Minor comment: insert "a" in. a comment.

**Review by luc-guyot-infomaniak — APPROVED** [review]

**Pull request merged**